src/common/components/WalletBalanceInput/WalletSelectorMenu.tsx

```diff
@@ -0,0 +1,77 @@
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Button,
+  HStack,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  VStack,
+} from '@chakra-ui/react';
+
+import { Text } from '../Typography';
+
+export interface ITokenBalance {
+  symbol: string;
+  amount: string;
+  icon: React.ReactNode;
+}
+
+interface WalletBalanceInputProps {
+  selectedWallet: ITokenBalance;
+  tokens: ITokenBalance[];
```

Comment:
Same as above, could we rename the `ITokenBalance` like the prop name because it's more understandable

```suggestion
export interface Token {
  symbol: string;
  amount: string;
  icon: React.ReactNode;
}

interface WalletBalanceInputProps {
  selectedToken: Token;
  tokens: Token[];
```

Or you can rename it to `Wallet` 🤔

Reply:
yeah let me renames these